src/test/skript/tests/syntaxes/expressions/ExprTimeSince.sk

```diff
@@ -0,0 +1,71 @@
+# Since code takes time to run, we need to give some leeway to the values.
+# Precision to within a tick should be more than enough.
+local function isRoughlyEqual(timespan1: timespan, timespan2: timespan) :: boolean:
+	set {_diff} to difference between {_timespan1} and {_timespan2}
+	if {_diff} is less than 1 tick:
+		return true
+	return false
+
+test "time since":
+
+	# normal dates
+	set {_time} to time since now
+	assert isRoughlyEqual({_time}, 0 seconds) is true with "Time since now failed (expected 0 seconds, got %{_time}%)"
+
+	set {_date} to 1 day ago
+	set {_time} to time since {_date}
```

Comment:
any reason for `{_date}` over `time since 1 day ago`?

Reply:
No, I think i wrote it like that in case i wanted the date in the assertion message.